src/app/marketplace/view/view-ad.component.html

```diff
@@ -74,6 +65,18 @@
             {{ data.productNumber }}
           </label-value>
         </ng-container>
+        <label-value *ngIf="ad.price" [label]="i18n.ad.price"
+          class="totalAmount">
+          <ng-container *ngIf="ad.promotionalPrice">
```

Comment:
This is a good candidate for using `*ngIf="ad.promotionalPrice; else plainPrice"`
And then
```html
<ng-template #plainPrice>
  {{ ad.price | currency:ad.currency }}
</ng-template>
```

Of course, this is just a cosmetic change.